components/illustratedmessage/stories/illustratedmessage.stories.js

```diff
@@ -88,3 +102,15 @@ WithForcedColors.parameters = {
 		modes: disableDefaultModes
 	},
 };
+/**
+ * Illustrated message comes in three sizes: small, medium, and large.
+ * - Small illustrated message is typically used in quick or in-line actions.
+ * - The medium size is the default, and often used in panels.
+ * - The large size illustrated message is generally used in full page layouts and dialogs.
+ */
+export const Sizing = SizingTemplate.bind({});
```

Comment:
Is it possible to use the `Sizes()` function in the decorators for this? I think it might be available in this branch now.